Annotation/declarative_golden_set.json

```diff
@@ -1422,7 +1532,19 @@
                     }
                 ],
                 "type": "declarative",
-                "clause": "",
+                "clause": "you revisit this page",
+                "embedded clauses": []
+            },
+            {
+                "predicate": [
+                    {
+                        "str": "ensure",
+                        "lemma": "",
+                        "POS": ""
+                    }
+                ],
+                "type": "declarative",
+                "clause": "re\u2013check for any updates/changes before the date of the fixture",

```

Comment:
Im not sure this is coordinated at the EC level, I think its coordinated after the pronoun "you" and therefore should be one EC rather than two